js/reference.js

```diff
@@ -36,7 +36,13 @@
 
         var defaultConfig = {
             internalHosts: {"type": "array", "value": []},
-            disableExternalLinkModal: {"type": "boolean", "value": false}
+            disableExternalLinkModal: {"type": "boolean", "value": false},
+            // NOTE: will be expanded later to include `closed` and `hidden`
+            facetPanelDisplay: {"type": "object", "value": {
```

Comment:
I don't think you should define the default here. It should be in chaise instead.